components/downstate/metadata/downstate.yml

```diff
@@ -0,0 +1,82 @@
+name: Down state
```

Comment:
We'll need to add the max-perspective token setup here once it's available, but design hasn't defined the token yet and therefore it's not in the token system at this time.